src/components/EventDetails.js

```diff
@@ -1,27 +1,28 @@
 import React from 'react';
 import { Button } from './Button';
 
-export const EventDetails = (event) => {
+export const EventDetails = (props) => {
   return(
     <div className='event-details-card'>
       <div className="event-details-container">
-        <h3>{event.name.text}</h3>
-        <img alt="Event" src={event.logo.url} />
+        <h3>{props.event.name.text}</h3>
+        <img alt="Event" src={props.event.logo.url} />
```

Comment:
Yes, you're right. I think we should add a check to see if we have a logo before displaying it. I overlooked that when I made the `EventDetails` component. According to the docs, the logo is optional: https://www.eventbrite.ca/developer/v3/response_formats/event/#ebapi-std:format-event